templates/inspec/examples/google_spanner_instance/google_spanner_instance.erb

```diff
@@ -0,0 +1,11 @@
+<% gcp_project_id = "#{external_attribute('gcp_project_id', doc_generation)}" -%>
+<% spannerinstance = grab_attributes['spannerinstance'] -%>
+
+describe google_spanner_instance(project: <%= gcp_project_id -%>, name: <%= doc_generation ? "'#{spannerinstance['name']}'" : "spannerinstance['name']" -%>, config: <%= doc_generation ? "'#{spannerinstance['config']}'" : "spannerinstance['config']" -%>) do
+  it { should exist }
+  its('config') { should eq <%= doc_generation ? "'#{spannerinstance['config']}'" : "spannerinstance['config']" -%> }
```

Comment:
use `match` instead of `eq` to match as a regular expression. The result coming from the API is `projects/your-project/instanceConfigs/configName`, so `should match 'configName'` will work.

Reply:
let me try